stubs/uWSGI/uwsgi.pyi

```diff
@@ -0,0 +1,241 @@
+def snmp_decr_counter32(__oid_num: int, __decrement: int) -> _TrueOrNone: ...
+def snmp_decr_counter64(__oid_num: int, __decrement: int) -> _TrueOrNone: ...
+def snmp_decr_gauge(__oid_num: int, __decrement: int) -> _TrueOrNone: ...
+@overload
+def send_to_spooler(__mesage_dict: dict[bytes, bytes]) -> bytes | None: ...
+@overload
+def send_to_spooler(
+    *, spooler: bytes = ..., priority: bytes = ..., at: bytes = ..., body: bytes = ..., **kwargs: bytes
+) -> bytes | None: ...
+
+spool = send_to_spooler
+
+def set_spooler_frequency(__frequency: int) -> Literal[True]: ...
+def spooler_jobs() -> list[bytes]: ...
+def spooler_pid() -> int: ...
+def spooler_pids() -> list[int]: ...
+def spooler_get_task(__task_path: str) -> dict[bytes, bytes] | None: ...
+def call(__rpc_name: str, **args: bytes) -> bytes | None: ...
+def chunked_read(__timeout: int = 0) -> bytes: ...
+def chunked_read_nb() -> bytes: ...
+def cl() -> int: ...
+def close(__fd: int) -> None: ...
+def connect(__socket_name: str, timeout: int = 0) -> int: ...
+def connection_fd() -> int: ...
+def disconnect() -> None: ...
+def embedded_data(__name: str) -> bytes: ...
+def extract(__name: str) -> bytes | None: ...
+def farm_get_msg() -> bytes | None: ...
+def farm_msg(__farm_name: str, __message: str | bytes | ReadOnlyBuffer) -> None: ...
+def get_logvar(__key: str | bytes) -> bytes | None: ...
+def green_schedule() -> Literal[True]: ...
+def i_am_the_lord(__name: str) -> bool: ...
+def i_am_the_spooler() -> _TrueOrNone: ...
+def in_farm(__farm_name: str = ...) -> _TrueOrNone: ...
+def is_connected(__fd: int = -1) -> bool: ...
+def is_locked(__lock_num: int = 0) -> bool: ...
+def listen_queue(__id: int = 0) -> int: ...
+def lock(__lock_num: int = 0) -> None: ...
+def log(__logline: str) -> Literal[True]: ...
+def log_this_request() -> None: ...
+def logsize() -> int: ...
+def lord_scroll(__legion_name: str) -> bytes | None: ...
+def masterpid() -> int: ...
+def mem() -> tuple[int, int]: ...
+def metric_dec(__key: str, __decrement: int = 1) -> _TrueOrNone: ...
+def metric_div(__key: str, __divisor: int = 1) -> _TrueOrNone: ...
+def metric_get(__key: str) -> int: ...
+def metric_inc(__key: str, __increment: int = 1) -> _TrueOrNone: ...
+def metric_mul(__key: str, __factor: int = 1) -> _TrueOrNone: ...
+def metric_set(__key: str, __value: int = 1) -> _TrueOrNone: ...
+def metric_set_max(__key: str, __value: int = 1) -> _TrueOrNone: ...
+def metric_set_min(__key: str, __value: int = 1) -> _TrueOrNone: ...
+def micros() -> int: ...
+def mule_get_msg(signals: bool = True, farms: bool = True, buffer_size: int = 65536, timeout: int = -1) -> bytes: ...
+def mule_id() -> int: ...
+@overload
+def mule_msg(__mesage: str | bytes | ReadOnlyBuffer) -> bool: ...
+@overload
+def mule_msg(__mesage: str | bytes | ReadOnlyBuffer, __mule_id: int) -> bool: ...
+@overload
+def mule_msg(__mesage: str | bytes | ReadOnlyBuffer, __farm_name: str) -> bool: ...
+def offload() -> Literal[b""]: ...
```

Comment:
This has been commented out, since it presumably isn't necessary anymore, so it's a no-op, but if we want to be backwards compatible, we should probably annotate the arguments it was going to parse, which was `__filename: str, __len: int = 0`